runtime/gc_realtime/MemorySubSpaceMetronome.cpp

```diff
@@ -153,9 +153,9 @@ MM_MemorySubSpaceMetronome::systemGarbageCollect(MM_EnvironmentBase *env, U_32 g
 		sched->startGC(envRealtime);
 		sched->setGCCode(MM_GCCode(gcCode));
 		/* if we were triggered by rasdump, then the caller has already acquired exclusive VM access */
-		sched->continueGC(envRealtime, SYSTEM_GC_TRIGGER, 0, env->getOmrVMThread(), J9MMCONSTANT_EXPLICIT_GC_RASDUMP_COMPACT != gcCode);
+		sched->continueGC(envRealtime, SYSTEM_GC_TRIGGER, 0, (OMR_VMThread *)envRealtime->getOmrVMThread(), J9MMCONSTANT_EXPLICIT_GC_RASDUMP_COMPACT != gcCode);
```

Comment:
why this change necessary?

Reply:
Its clearly not necessary. Its the result of repeatedly rebasing and redoing my changes dozens of times.